tests/tests/datm_cdeps_lnd_era5

```diff
@@ -0,0 +1,58 @@
+#
+#  DATM_CDEPS_NOAHMP_ERA5 test
+#
+
+export TEST_DESCR="DATM_CDEPS_NOAHMP_ERA5 - control "
+
+export CNTL_DIR="datm_cdeps_lnd_era5"
```

Comment:
I see a few extraneous quotes in this test. Here on this line, and then on LN 25,26,32,38-42. The same changes need to be be made in the restart test. Thanks!

Reply:
@DeniseWorthen I'll fix it soon.